build.gradle.kts

```diff
@@ -21,7 +21,7 @@ repositories {
 }
 
 dependencies {
-    api("app.revanced:multidexlib2:2.5.2.r2")
+    implementation("app.revanced:multidexlib2:2.5.2.r2")
```

Comment:
How are opcodes done on the patches? Because now `Opcode.*` is inaccessible.

Reply:
The patcher is not responsible for that. It is the patch's purpose to use that dependency, note the usage of the patcher can be merely to merge integrations, and that's it where a dependency to revanced:multidexlib2  is not required. The commit is intended.

Reply:
Then people will have to update multidexlib accordingly with the patcher, which may not always be wanted. But your reasoning seems fair. Though documenting it is required. And we should refrain from updating multidexlib as much as possible then to prevent such inconveniences from happening. This is a opensource project, you should think about your fellow developers who may decide to use this.